build/testdata/051.bzl.golden

```diff
@@ -17,14 +17,17 @@ fff
 ggg
 
 def function(
-    # This shouldn't mess with indentation counting
-    x,
-    y = z,
-):
+        # This shouldn't mess with indentation counting
+        x,
+        y = z):
     aaa
     bbb
 
-    def g(a, s, d):
+    def g(
+            a,
+            s,
+            d  # this is d
```

Comment:
I wonder if we should have `d): # this is d` instead.

Reply:
This is currently not supported yet, but what if there are comments on both of these lines? (the line with the last argument and the line with `):`)

Reply:
On which node is the comment associated?

Reply:
In the example above, to `d`, and it's not yet supported to assign comments to block headers. But it's still possible that an input file looks like the following:

    def f(  # def beginning
        x  # my argument
    ):  # def end
        pass

In theory I can format it like this:

    def f(  # def beginning
            x):  # my argument
        # def end
        
        pass

Reply:
I've decided to go with your proposal. This formatting style is not final anyway and may be changed in the future.